tests/AccountFactoryECDSATest.py

```diff
@@ -0,0 +1,37 @@
+from LTO.Accounts.AccountFactoryECDSA import AccountECDSA as AccountFactory
+from LTO.tools import Tools
+
+
+class TestAccountECDSA():
+    factory = AccountFactory('L')
+    seed = 'curtain want coffee other panel satoshi tissue chief floor lumber size dinner upper verb wall'
+    account = factory.createFromSeed(seed)
+
+    def testMakeKey(self):
+        assert self.factory._MakeKey(self.seed).to_string() == b"\xa4\xd12%\xff\xb9\x89\xfc7 \t^\x97\xdd\x9a(r\x10'\x0f\x85\xca\x11Jb=Pn\xfd\xab\xb1\x1d"
+
+    def testCreateAddress(self):
+        assert self.factory.createAddress(self.account.publicKey) == self.account.address
+
+    def testCreateSignKeys(self):
+        privateKey, publicKey = self.factory.createSignKeys(self.seed)
+        assert self.account.publicKey == publicKey
+        assert self.account.privateKey == privateKey
+
+
+    def testCreateFromPublic(self):
+        account = self.factory.createFromPublicKey(self.account.publicKey)
+        Tools().__eq__(account, self.account)
```

Comment:
Are there assertions in this `Tools()` class? I guess it compares all the fields in 2 dicts?

Reply:
it returns false in case there's a difference between the two objects, then the test fails, no need for assertion in that case